ChestsPlusPlus_Main/src/main/java/com/jamesdpeters/minecraft/chests/filters/ItemTypeUtil.java

```diff
@@ -14,6 +19,24 @@ public class ItemTypeUtil {
     static {
         tags = new ArrayList<>();
 
+        tags.add(new Tag<>() {
```

Comment:
I would move this `Tag` into a new class called ChestsPlusPlusTag. And store it as a static Tag variable called SEEDS.

That way it can be used in other places if needed too called as:
`ChestsPlusPlusTag.SEEDS`

Reply:
Good idea, I'll do it. I saw your comment after I pushed the new stuff, so I'll add it after that